06_distributed_advanced.ipynb

```diff
@@ -209,7 +209,12 @@
    "metadata": {},
```

Comment:
<p>Unfortunately, I had to comment this out to avoid hanging while executing the notebook. I think that's an acceptable tradeoff.</p><p> </p><p>AFAICT, nbsphinx can't skip execution of individual cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>